frontend/src/config/index.js

```diff
@@ -0,0 +1,9 @@
+export const envVariables = {
+    development: {
+      baseURLAPI: "", //proxy for local development
+    },
+    production: {
+      baseURLAPI: "http://localhost:4000", // change this with the URL of the deployed BE
```

Comment:
Please remove the line `"proxy": "http://localhost:4000",` from `package.json` file.

Reply:
we still might need the proxies :)) - based on some reading i've done and because of CORS - I will do some more research and come back to this to put it in a better shape.

Reply:
@danalexandrunicoara 
I've removed the proxy from package.json and added a cors package on BE to allow all CORS for now until I will create a production and development environement on BE. Give a thumbs up in a comment if everything is fine and I will merge this. Thank you! Cheers!